src/engine/include/sourcemeta/includejs/engine_value.h

```diff
@@ -24,6 +25,9 @@ class SOURCEMETA_INCLUDEJS_ENGINE_EXPORT Value {
   Value(Value &&other) noexcept;
   Value(const Value &other) = delete;
 
+  using Function = std::function<Value(std::span<Value> arguments)>;
+  using FunctionStorage = std::unordered_map<void *, Value::Function>;
```

Comment:
I don't think `Value::Function` is valid. I think you want just `Function`?

Reply:
Both don't work. I'll check again.

Reply:
For instance I have the same Error with `std::span` that is properly include at the top of the file haha

Reply:
```
/Users/tonygorez/oss/includejs/src/engine/include/sourcemeta/includejs/engine_value.h:28:45: error: no member named 'span' in namespace 'std'
  using Function = std::function<Value(std::span<Value> arguments)>;
                                       ~~~~~^
```

Reply:
All worked nicely there: https://github.com/sourcemeta/includejs/compare/set-function?expand=1

I wonder where I fucked up I'll investigate more